dotnet-base-libraries/CryptographyRandomNumberGenerator/App/CryptographicHelpers.cs

```diff
@@ -0,0 +1,16 @@
+using System.Security.Cryptography;
+
+namespace App;
+
+public static class CryptographicHelpers
+{
+    public static byte[] GenerateRandomKey(int bytesCount)
+    {
+        return RandomNumberGenerator.GetBytes(bytesCount);
+    }
+
+    public static int GenerateRandomInteger(int minValue, int maxValue)
+    {
+        return RandomNumberGenerator.GetInt32(minValue, maxValue);
```

Comment:
We can make this an expression body statement

Reply:
@jshergal Done